hello-world-master/README.md

```diff
@@ -1,2 +1,46 @@
-# hello-world
-A basic sample channel application for Roku
+# Channel application for Roku
+
+## Use Case
+
+This project is a Roku sample app designed as a base for experimenting with Scene Graph architecture and BrightScript features. 
+
+## How to Run This Sample
+* Zip the contents of the main file of this project and deploy to your roku device. Follow the developer set-up guide here for a quick guide on how to do so: https://developer.roku.com/en-ca/docs/developer-program/getting-started/developer-setup.md
```

Comment:
Just a small change... you can link a phrase using `[]()` [example](www.google.com)
You can use this on "developer set-up guide"